sklearn/tests/test_cross_validation.py

```diff
@@ -1038,3 +1039,57 @@ def test_check_is_partition():
 
     p[0] = 23
     assert_false(cval._check_is_partition(p, 100))
+
+
+def test_disjoint_label_folds():
+    """ Check that the function produces equilibrated folds
+        with no label appearing in two different folds
+    """
```

Comment:
Could you also convert this to a comment following #4432 ? (sorry for the trouble)